endpoints/data_objects/src/main.cpp

```diff
@@ -306,6 +309,143 @@ namespace irods::http::handler
 
 namespace
 {
+	//
+	// Utility functions
+	//
+
+#ifdef IRODS_HTTP_API_INC_READ_IMPL_0
```

Comment:
Everything related to the simplified read op is still a WIP. Both implementations work, but I'm investigating different designs/implementations.

Reply:
Done.